rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -353,7 +369,18 @@ def test_upgrade_v4_v5(globaldb):
             nodes_tuples_from_db = cursor.execute('SELECT * FROM default_rpc_nodes').fetchall()
             assert nodes_tuples_from_db == nodes_tuples_from_file
 
-        last_queried_ts = globaldb_get_general_cache_last_queried_ts_by_key(
+        # check again existence of unique_cache table
+        cursor.execute(
+            'SELECT COUNT(*) FROM sqlite_master WHERE type="table" and name=?',
+            ('unique_cache',),
+        )
+        assert cursor.fetchone()[0] == 1
+        value = globaldb_get_cache_values(
+            cursor=cursor,
+            key_parts=['TEST_KEY', '123'],
+        )
+        assert value == ['klm']
```

Comment:
As before I wouldn't test this here. We had a function here because we are making changes to the curve cache but is a different topic